### ISSUE TITLE
In windows I'm unable to install the dependencies. mainly  mxnet and gluoncv

### ISSUE THREAD

**Issue by alan-ai-learner** [OPHoperHPO/image-background-remove-tool#44]

**Comment by OPHoperHPO** [comment]
Hi, @alan-ai-Learner
You can visit the [gluoncv website](https://cv.gluon.ai/install.html) for instructions on how to install these libraries on Windows.

**Comment by Extent421** [comment]
I ran into the same thing,  the gluoncv website also doesn't work, as their instructions point to releases that don't exist, at least for python 3.9.

I was able to get it to build by installing modules one at a time.  Pip complains about a lot of broken dependencies but it seems to work fine.  I ended up with the following packages

numpy 1.16.6
tqdm 4.61.2
torch 1.9.0
torchvision 0.10.0
opencv_contrib_python 4.5.3.56
scipy 1.7.0
tensorflow 2.5.0
gluoncv 0.10.3
gdown 3.13.0
scikit_image 0.18.2
Pillow 8.3.1
mxnet 1.7.0.post2
